boards/st/nucleo_l476rg/nucleo_l476rg.dts

```diff
@@ -193,3 +194,10 @@ stm32_lp_tick_source: &lptim1 {
 		};
 	};
 };
+
+&rng {
+	/* Change clock source to avoid using MSI */
+	clocks = <&rcc STM32_CLOCK(AHB2, 18U)>,
+		 <&rcc STM32_SRC_PLL_Q CLK48_SEL(2)>;
```

Comment:
If the RNG clock @ 48MHz is sourced by PLL Q , its freq must be 48MHz or less and with the rcc node configuration node above, div-q = <2>; the PLL Q is 160MHz